<launch>
  <!-- We resume the logic in empty_world.launch, changing only the name of the world to be launched -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find targets_path_planning)/worlds/heightmap.world"/>
    <arg name="paused" value="false"/>
  </include>
  <node name="target_assignment" pkg="targets_path_planning" type="target_assignment.py" output="screen" />
  <node name="paths_getter" pkg="targets_path_planning" type="paths_getter.py" output="screen" />
  <node name="poses_getter" pkg="targets_path_planning" type="poses_getter.py" output="screen" />
</launch>
